src/popup/pages/AboutUs/index.js

```diff
@@ -83,6 +83,9 @@ class AboutUs extends React.Component {
           {this.renderTopInfo()}
           {this.renderWalletDetail()}
         </div>
+        <div className="about-tip-container">
+          <p className="about-tip">Powered by Bit Cat and Simply VC</p>
```

Comment:
Could you link these two to:
- https://www.bitcat365.com/en/
- https://simply-vc.com.mt/

Reply:
ok I'll try adding that to this pr

Reply:
turns out it could use some design review. I'll put it in a separate PR.